inspire/base/recordext/fields/inspire.cfg

```diff
@@ -519,6 +519,8 @@ title_translation:
 
 
 url:
+    schema:
+        {'url': {'type': 'list', 'force': True}}
```

Comment:
@pedrogaudencio As witnessed in #196, be careful when changing the type of the field here as existing submissions probably have the URL value as string.